addons/account_check_printing/models/account_payment.py

```diff
@@ -69,14 +69,20 @@ def _constrains_check_number(self):
               JOIN account_journal journal ON journal.id = move.journal_id,
                    account_payment other_payment
               JOIN account_move other_move ON other_move.id = other_payment.move_id
-             WHERE payment.check_number::BIGINT = other_payment.check_number::BIGINT
                AND move.journal_id = other_move.journal_id
```

Comment:
The Where statement has now disappeared, you should add it back
```suggestion
               WHERE move.journal_id = other_move.journal_id
```